tests/wp-cli/test-vip-data-cleanup.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Automattic\VIP\Tests;
+
+use WP_UnitTestCase;
+
+require_once __DIR__ . '/../../vip-helpers/vip-wp-cli.php';
+require_once __DIR__ . '/../../wp-cli/vip-data-cleanup.php';
+
+// phpcs:ignore PEAR.NamingConventions.ValidClassName.Invalid
+class VIP_Data_Cleanup_Command__Test extends WP_UnitTestCase {
+	private $original_srtm;
+	private $srtm_existed;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		// Backup the srtm property state
+		global $wpdb;
+		$this->srtm_existed = property_exists( $wpdb, 'srtm' );
+		if ( $this->srtm_existed ) {
+			$this->original_srtm = $wpdb->srtm;
+		}
+	}
+
+	public function tearDown(): void {
+		// Restore the srtm property state
+		global $wpdb;
+		if ( $this->srtm_existed && property_exists( $wpdb, 'srtm' ) ) {
+			$wpdb->srtm = $this->original_srtm;
+		}
```

Comment:
```suggestion
		if ( $this->srtm_existed ) {
			$wpdb->srtm = $this->original_srtm;
		}
```

Even if the property does not exist now, we should restore it to what it used to be.

Reply:
We may want to add an `elseif` block: if the property exists now, unset it.